components/common/src/command/package/install.rs

```diff
@@ -807,14 +807,29 @@ impl<'a> InstallTask<'a> {
         ident: &FullyQualifiedPackageIdent,
         artifact_path: &Path,
     ) -> Result<()> {
-        let cache_path = self.cached_artifact_path(ident);
+        // Canonicalize both paths to ensure that there aren't any symlinks when comparing them
+        // later.
+        let artifact_path = artifact_path.canonicalize()?;
+        let cache_path = self.cached_artifact_path(ident).canonicalize()?;
```

Comment:
![tenor-48006762](https://user-images.githubusercontent.com/207178/37783346-a62e44cc-2dcb-11e8-9a79-492f60d71e9a.gif)
... to canonicalize both paths!

Reply:
Figured I'd go full paranoid here